python-package/lightgbm/basic.py

```diff
@@ -4146,8 +4151,13 @@ def __boost(
 
         Returns
         -------
-        is_finished : bool
-            Whether the boost was successfully finished.
+        produced_empty_tree : bool
+            ``True`` if the tree(s) produced by this iteration did not have any splits.
+            This usually means that training is "finished" (calling ``update()`` again
+            will not change the model's predictions). However, that is not always the
+            case. For example, if you have added any randomness (like column sampling by
+            setting ``feature_fraction_bynode < 1.0``), it is possible that another call
+            to ``update()`` would produce a non-empty tree.
```

Comment:
I think it was a copy-paste mistake.
```suggestion
        produced_empty_tree : bool
            ``True`` if the tree(s) produced by this iteration did not have any splits.
            This usually means that training is "finished" (calling ``__boost()`` again
            will not change the model's predictions). However, that is not always the
            case. For example, if you have added any randomness (like column sampling by
            setting ``feature_fraction_bynode < 1.0``), it is possible that another call
            to ``__boost()`` would produce a non-empty tree.
```

Reply:
good catch!